src/reducers/scrollStateReducer.js

```diff
@@ -48,10 +49,10 @@ function scrollStateReducer(state = DEFAULT_STATE, action) {
   switch (action.type) {
     case ActionTypes.INITIALIZE:
       let { props } = action;
-
       state = updateRowCount(state, props);
       state = updateRowHeights(state, props);
       state = updateViewHeight(state, props);
+      state = updateVisibleRows(state);
```

Comment:
I'm assuming these methods aren't independent.  Given that, what's a good way to keep track of their interdependencies?  Can we add a comment stating when updateVisibleRows needs to be called and what helper methods it needs to be called after and why.